custom_components/tuya_local/devices/ecostrad_iq_heating_element.yaml

```diff
@@ -0,0 +1,116 @@
+name: Ecostrad iQ Heating Element
+primary_entity:
+  entity: climate
+  icon: "mdi:radiator"
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: true
+          value: "heat"
+        - dps_val: false
+          value: "off"
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: "auto"
+          value: Program

```

Comment:
Where possible use standard preset names (none, eco, away, boost, comfort, home, sleep, activity) and extended ones defined in one of the climate classes in translations/en.json (manual, program, temp_override, perm_override), (cool, fan_only, heat, floor_cool, floor_heat) or (smart_heat, quick_heat, quiet_heat, smart_cool. quick_cool, quiet_cool, auto).

Note these are case sensitive, so should be in all caps, and if you use any extended ones, you need to set the class of the entity appropriately.